apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx

```diff
@@ -50,15 +55,9 @@ const AITutorPanel: React.FunctionComponent<AITutorPanelProps> = ({open}) => {
     },
   ];
 
-  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
-    e.preventDefault();
-    dispatch(addAIResponse(''));
-    setSelected(e.target.value);
-  };
-
```

Comment:
@Erin007 I wasn't 100% sure about removing the dispatch line but I figured we could probably move that functionality somewhere else if it's still needed now that the onChange doesn't apply

Reply:
I think that's totally fine. The dispatch line was just setting the AI's response to an empty string, which you're right, we'll end up doing somewhere else if we need it.